app/src/main/java/com/example/concordia_campus_guide/Helper/POIGridView.java

```diff
@@ -0,0 +1,39 @@
+package com.example.concordia_campus_guide.Helper;
+
+import android.util.AttributeSet;
+import android.widget.GridView;
+import android.content.Context;
+
+public class POIGridView extends GridView {
+
+    public POIGridView(Context context) {
+        super(context);
+    }
+
+    public POIGridView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public POIGridView(Context context, AttributeSet attrs, int defStyle) {
+        super(context, attrs, defStyle);
+    }
+
+    @Override
+    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
```

Comment:
Can you add a comment for this function? I don't understand what it's needed for

Reply:
done